api/v2/checluster_types.go

```diff
@@ -155,6 +155,9 @@ type CheClusterDevEnvironments struct {
 	// If not specified, the pod scheduler is set to the default scheduler on the cluster.
 	// +optional
 	PodSchedulerName string `json:"podSchedulerName,omitempty"`
+	// RuntimeClassName specifies the spec.runtimeClassName for workspace pods.
+	// +optional
+	RuntimeClassName *string `json:"runtimeClassName,omitempty"`
```

Comment:
I've set the type to `*string` because that is the type for `RuntimeClassName` in `PodSpec` struct:

https://github.com/kubernetes/api/blob/9dc1a952bda9aeb400aa6409b7d89eff50d6ea77/core/v1/types.go#L3920

Reply:
Maybe `PodRuntimeClassName` ?

Reply:
IMHO I don't know if it's necessary since we have names such as:
https://github.com/eclipse-che/che-operator/blob/813dcbf0b42bc766421874f4cd9b3e7af61453f5/api/v2/checluster_types.go#L95-L100